custom/icds_reports/models/aggregate.py

```diff
@@ -1688,3 +1690,51 @@ class Meta(object):
 
     _agg_helper_cls = AggServiceDeliveryReportHelper
     _agg_atomic = False
+
+
+class AggBiharAPIDemographics(models.Model, AggregateMixin):
+    state_id = models.TextField(null=True)
```

Comment:
why all these duplicated fields if we are joining with awc_location anyway?

Reply:
Yeah this is because the query will be filtered on location_ids and this table will have index n them so it should increase the speed of query. Because I tried making a view of joining awc_location and this table when view uses state_id of awc_location the index is not used.

Reply:
Can you explain this a bit more, where would the index help?